src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -209,7 +209,18 @@ MOBILE_INTEGRATION_TEST_TOKEN = '{{ MOBILE_INTEGRATION_TEST_TOKEN }}'
 
 ####### For Constructing Global URLs ########
 
-LOCAL_APPS = ('dimagi.utils', 'gunicorn',)
+LOCAL_APPS = (
+    'dimagi.utils',
+    'gunicorn',
+    {% for app_name, host in localsettings.get('LOCAL_APPS', {}).items() %}
+    {% if host == inventory_hostname %}
+    '{{ app }}',
+    {% endif %}
+    {% endfor %}
+)
+ {% if 'DATADOG_TRACE' in localsettings %}
```

Comment:
nit: extra leading whitespace?

Reply:
This should be addressed now!

Reply:
did you forget to push a commit? there's still a leading space (not a big deal, just pointing out that it doesn't appear to be fixed)